samples/language/ConditionalBranching.qs

```diff
@@ -33,9 +33,8 @@ namespace MyQuantumApp {
             Message("It is way too hot");
         }
 
-        let fahrenheit = 40;
```

Comment:
Do we want to keep this one

Reply:
I think it might be there to make it easier for someone to change it to a different value? I would actually keep it but make it -40 so that absolute value is more interesting

Reply:
Can you please revert the line removal and make it -40, so that it's more clear what's going on here?